src/passes/references/expectedLocationAnalyser.ts

```diff
@@ -320,7 +319,7 @@ export class ExpectedLocationAnalyser extends ASTMapper {
     this.visitStatement(node, ast);
   }
 
-  visitCairoAssert(node: CairoAssert, ast: AST): void {
```

Comment:
You've removed `visitCairoAssert` in `visitor.ts` so the function won't be called